### DISCUSSION TITLE
Play Video from Uploaded File

### DISCUSSION THREAD

**Discussion by leimao** [ppoffice/hexo-theme-icarus#1014] [Q&A] [answered]
Hello, 
I was wondering, instead of uploading the video to YouTube and insert the video in the post, whether we can play video directly from an uploaded (mp4) video file? Thank you.

**Comment by ppoffice** [comment, marked as answer]
https://www.w3schools.com/html/html5_video.asp

**Reply by leimao** [reply]
Thank you.
